systems/notification/event-notify/pkg/server/event.go

```diff
@@ -52,684 +52,743 @@ func (es *EventToNotifyEventServer) EventNotification(ctx context.Context, e *ep
 		if err != nil {
 			return nil, err
 		}
-		
-		jmsg, err := json.Marshal(msg)
-		if err != nil {
-			log.Errorf("Failed to store raw message for %s to db. Error %+v", c.Name, err)
-		}
-		
-		dynamicConfig := c
-		shortNodeId := msg.NodeId
-		if len(msg.NodeId) > 6 {
-			shortNodeId = msg.NodeId[len(msg.NodeId)-6:]
-		}
-		
-		dynamicConfig.Title = fmt.Sprintf("Node %s: %s", shortNodeId, msg.State)
-		dynamicConfig.Description = fmt.Sprintf("Status: %s", msg.Substate)
-		
-		notificationType := notif.TYPE_INFO
-		
-		if msg.State == "Faulty" {
-			notificationType = notif.TYPE_CRITICAL
-		} else if msg.State == "Unknown" {
-			notificationType = notif.TYPE_ACTIONABLE_WARNING 
-		}
-		
-		if notificationType == notif.TYPE_INFO {
-			switch msg.Substate {
-			case "off":
-				notificationType = notif.TYPE_WARNING
-			case "reboot", "update", "upgrade", "downgrade":
-				notificationType = notif.TYPE_WARNING
-			}
-		}
-		
-		dynamicConfig.Type = notificationType
-		
-		_ = es.ProcessEvent(&dynamicConfig, es.orgId, "", msg.NodeId, "", "", jmsg, msg.NodeId)
-		
+		handleEventNodeStateTransition(es, msg, &c)
+
 	case msgbus.PrepareRoute(es.orgName, evt.EventRoutingKey[evt.EventPaymentSuccess]):
 		c := evt.EventToEventConfig[evt.EventPaymentSuccess]
 		msg, err := epb.UnmarshalPayment(e.Msg, c.Name)
 		if err != nil {
 			return nil, err
 		}
+		handleEventPaymentSuccess(es, msg, &c)
 
-		jmsg, err := json.Marshal(msg)
+	case msgbus.PrepareRoute(es.orgName, evt.EventRoutingKey[evt.EventPaymentFailed]):
+		c := evt.EventToEventConfig[evt.EventPaymentFailed]
+		msg, err := epb.UnmarshalPayment(e.Msg, c.Name)
 		if err != nil {
-			log.Errorf("Failed to store raw message for %s to db. Error %+v", c.Name, err)
+			return nil, err
 		}
+		handleEventPaymentFailed(es, msg, &c)
 
-		if msg.ItemType != ukama.ItemTypeInvoice.String() {
-			log.Errorf("unexpected item type for successful payment: %s", msg.ItemType)
-			break
+	case msgbus.PrepareRoute(es.orgName, evt.EventRoutingKey[evt.EventInvoiceGenerate]):
+		c := evt.EventToEventConfig[evt.EventInvoiceGenerate]
+		msg, err := epb.UnmarshalInvoiceGenerated(e.Msg, c.Name)
+		if err != nil {
+			return nil, err
 		}
+		handleEventInvoiceGenerate(es, msg, &c)
 
-		metadata := map[string]string{}
+	default:
+		log.Errorf("No handler routing key %s", e.RoutingKey)
+	}
 
-		err = json.Unmarshal(msg.Metadata, &metadata)
-		if err != nil {
-			log.Errorf("failed to Unmarshal payment metadata as map[string]string: %v", err)
-		}
+	return &epb.EventResponse{}, nil
+}
 
-		targetId, ok := metadata["targetId"]
-		if !ok {
-			log.Errorf("missing targetId metadata for successful package payment: %s", msg.ItemId)
-			break
-		}
+func handleEventOrgAdd(es *EventToNotifyEventServer, msg *epb.EventOrgCreate, c *evt.EventConfig) {
+	jmsg, err := json.Marshal(msg)
+	if err != nil {
+		log.Errorf("Failed to store raw message for %s to db. Error %+v", c.Name, err)
```

Comment:
Almost all of these marshalling errors logging 
`		log.Errorf("Failed to store raw message for %s to db. Error %+v", c.Name, err)`
are missleading, because the error here is just a marshaling error, not a failure to store something in DB.
So, check for all of these.

Reply:
addressed